packages/fxa-settings/src/pages/WebChannelExamples/en.ftl

```diff
@@ -0,0 +1,8 @@
+## Legal page. This page contains simply a header and links to pages that display
+## content from https://github.com/mozilla/legal-docs
+
+legal-header = Legal
+# Links to our internal "Firefox Cloud" /legal/terms page
+legal-terms-of-service-link = Terms of Service
+# Links to our internal "Firefox Cloud" /legal/terms page
+legal-privacy-link = Privacy Notice
```

Comment:
These three strings/IDs already exist. It looks like this file was copied and pasted from [`packages/fxa-settings/src/pages/Legal/en.ftl` ](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/pages/Legal/en.ftl). Was this not a shared component?

Either way, Fluent strings get bundled together so we'll have strings with duplicate IDs if we use the same IDs here. If we do need to create this extra file, then I recommend updating the IDs to include its location so something like `web-channel-examples-legal-header`.

Reply:
This was a mistake. This is a developer only page. So no need for translation.